index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/5e67ba493b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="nav">
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="socials">
+          <ul>
+            <li>
+              <a href="https://www.facebook.com/"
+                ><img src="./assets/facebook.png" alt="facebook logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/nerdy_me_"
+                ><img src="./assets/twitter.png" alt="twitter logo"
+              /></a>
+            </li>
+            <li><a href="">English</a></li>
+            <li><a href="">MY Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+        <div class="more">
+          <div class="main-logo">
+            <a href="./index.html"
+              ><img src="./assets/summit logo.png" alt="summit logo"
+            /></a>
+          </div>
+          <div class="page-links">
+            <ul>
+              <li><a href="./about.html" target="_blank">About</a></li>
```

Comment:
- [OPTIONAL ] I think it would be better to prevent reopening a new tab when a user navigates to another page for a good user experience.